bin/faucet/src/faucet/mod.rs

```diff
@@ -415,21 +404,6 @@ impl Faucet {
         Ok((account_delta, block_number, notes, tx_id))
     }
 
-    /// Compiles the transaction script that creates the given set of notes.
-    fn compile(&self, notes: &[Note]) -> MintResult<TransactionArgs> {
-        let partial_notes = notes.iter().map(Into::into).collect::<Vec<_>>();
-        let script = self
-            .account_interface
-            .build_send_notes_script(&partial_notes, None, false)
-            .map_err(MintError::ScriptCompilation)?;
```

Comment:
Does the faucet still work without this tx script, or is that added somewhere else?